docs/source/examples/Notebook/rstversions/Typesetting Equations.rst

```diff
@@ -50,7 +50,7 @@ Source
     \end{equation*}
 
 Display
--------
+~~~~~~~
 
```

Comment:
Although the math directive is a Sphinx extension thing, not a normal part of rst. So maybe we should split out a `markdown2sphinx_rst` filter for that. In practical use, though, I think Sphinx is the main reason people want rst.